tests/test_processing_chain.py

```diff
@@ -248,3 +248,21 @@ def test_output_types(spms_raw_tbl):
     assert isinstance(lh5_out["wf_out"], lgdo.WaveformTable)
     assert isinstance(lh5_out["aoa_out"], lgdo.ArrayOfEqualSizedArrays)
     assert isinstance(lh5_out["vov_max_out"], lgdo.VectorOfVectors)
+
+
+def test_output_attrs(geds_raw_tbl):
+    dsp_config = {
+        "outputs": ["wf_blsub"],
+        "processors": {
+            "wf_blsub": {
+                "function": "bl_subtract",
+                "module": "dspeed.processors",
+                "args": ["waveform[0:100]", "baseline", "wf_blsub"],
+                "unit": "ADC",
+                "lh5_attrs": {"test_attr": "This is a test"},
```

Comment:
Can you please change it like this?

```suggestion
                "lgdo_attrs": {
                  "description": "This is a test",
                  "hdf5_settings": {
                    "compression": "gzip",
                    "shuffle": True,
                  }
                },
```

Reply:
I added this, but it won't actually compress anything without writing the object (and writing won't work for this since it's a WaveformTable). If we want to include a test of writing using this, it would probably be best as part of the integration tests